library/src/c_api.rs

```diff
@@ -254,7 +254,10 @@ pub extern "C" fn shorebird_report_launch_success() {
 #[cfg(test)]
 mod test {
     use super::*;
-    use crate::network::{testing_set_network_hooks, PatchCheckResponse};
+    use crate::{
+        network::{testing_set_network_hooks, PatchCheckResponse},
```

Comment:
I don't know what preferred rust style is, but I've tended to just split these out into two separate lines.
use crate::network::{testing_set_network_hooks, PatchCheckResponse};
use crate::platform;

Reply:
🤷  I just did what the formatter/vscode put in there. Will update.

Reply:
It looks like it's still in an unstable state, but `rustfmt` has an option to [do this for us](https://rust-lang.github.io/rustfmt/?version=v1.4.38&search=group#imports_granularity). Once that's stable, we should add it to `rustfmt.toml`.